db/migrations/sql/V2019.07.29.1316__InitialSchema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE users
```

Comment:
I think we can add other columns here like FirstName and LastName which would be applied to the profile as attributes